tests/e2e-leg-12/scale-up-with-auth-tls-keda/45-create-vas.yaml

```diff
@@ -0,0 +1,35 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
+kind: VerticaAutoscaler
+metadata:
+  name: v-scale-up-auth-tls-keda-vas
+spec:
+  verticaDBName: v-scale-up-auth-tls-keda
+  serviceName: pri1
+  scalingGranularity: Pod
+  customAutoscaler:
+    type: ScaledObject
+    scaledObject:
+      minReplicas: 3
+      maxReplicas: 5
+      metrics:
+      - name: vertica_sessions_running_total
+        metricType: AverageValue
+        authSecret: prometheus-tls
```

Comment:
This secret prometheus-tls was created under the namespace Prometheus in step 35. Should it be under the same namespace as VDB?